### PR TITLE
fix(cluster): vip dnat to k8s-svc-ip before kube-proxy running

### PR DESCRIPTION
**Pull request by kingofstormland**
/kind bug
fix  https://github.com/tkestack/tke/issues/606

### CODE REVIEW

File: pkg/platform/provider/baremetal/phases/keepalived/keepalived.go [hunk 1]
```diff
@@ -151,9 +151,20 @@ func ClearLoadBalance(s ssh.Interface, vip string, kubernetesSvcIP string) {
...
-	cmd := fmt.Sprintf("iptables -t nat -C %s -d %s -p tcp --dport 6443 -j DNAT --to-destination %s:443",
+	cmd := fmt.Sprintf("ip a | grep %s", option.KubernetesSvcIP)
+	stdout, _, exit, err := s.Exec(cmd)
+	// make sure svcPortChain name exists in iptables
+	if err != nil || exit != 0 {
+		return fmt.Errorf("exec %q failed:exit %d:error %s:stdout %s", cmd, exit, err, stdout)
+	}
+
+	if !strings.Contains(stdout, option.KubernetesSvcIP) {
+		return fmt.Errorf("ipvs is not installed yet k8s svc ip:%s", option.KubernetesSvcIP)
```
Comment (davidwtf) [inline pkg/platform/provider/baremetal/phases/keepalived/keepalived.go:162]: should not return error

Comment (davidwtf) [reply pkg/platform/provider/baremetal/phases/keepalived/keepalived.go:162]: iptables mode has similar problem

**Comment by kingofstormland** [comment]
in some case kube-proxy is not  runnabled, so loadbalance will be skipped until kube-proxy is running

**Pull request merged**